extensions/sql-bindings/src/common/constants.ts

```diff
@@ -25,7 +56,7 @@ export const output = localize('output', "Output");
 export const selectBindingType = localize('selectBindingType', "Select type of binding");
 export const selectAzureFunction = localize('selectAzureFunction', "Select an Azure function in the current file to add SQL binding to");
 export const sqlTableOrViewToQuery = localize('sqlTableOrViewToQuery', "SQL table or view to query");
-export const sqlTableToUpsert = localize('sqlTableToUpsert', "SQL table to upsert into");
+export const sqlTabconstoUpsert = localize('sqlTabconstoUpsert', "SQL table to upsert into");
```

Comment:
Mistype?

Reply:
Copied from original code. Will change to sqlTableToUpsert